src/site/content/en/blog/advancing-framework-ecosystem-cds-2019/index.md

```diff
@@ -138,10 +134,8 @@ Next.js, such as:
     surface better errors and warnings ([RFC](https://github.com/zeit/next.js/issues/9310)).
 
 <figure class="w-figure">
-  <img src="conformance.png" class="w-screenshot-filled" alt="Example of a conformance build error in Next.js">
-  <figcaption class="w-figcaption">
-    An example of a conformance build error in Next.js (prototype)
-  </figcaption>
+  {% Img src="image/admin/LoKCHqIIpGkQIUjxZlre.png", alt="Example of a conformance build error in Next.js", width="800", height="367", class="w-screenshot-filled" %}
```

Comment:
same note as above. If we fix it we should check to see if it looks ok. The filled class can cause some images to look squished on mobile.